devito/symbolics/extended_dtypes.py

```diff
@@ -1,10 +1,13 @@
+import ctypes as ct
```

Comment:
For consistency with the rest of the codebase: `import ctypes` rather than `import ctypes as ct`